onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -276,15 +287,25 @@
   qnn_backend_manager_ = std::make_unique<qnn::QnnBackendManager>(
       std::move(backend_path),
       profiling_level,
-      rpc_control_latency,
-      htp_performance_mode,
       context_priority,
       std::move(qnn_saver_path),
-      device_id,
+      device_id_,
       htp_arch,
       soc_model);
 }
 
+QNNExecutionProvider::~QNNExecutionProvider() {
+  // clean up thread local context caches
+  {
```

Comment:
why extra {} ?

Reply:
will remove it.

---
In reply to: [1491940197](https://github.com/microsoft/onnxruntime/pull/19521#discussion_r1491940197) [](ancestors = 1491940197)